app/controllers/sulten/lyche_opening_hours_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Sulten::LycheOpeningHoursController < ApplicationController
+  def index
+    @lyche_opening_hours= Sulten::LycheOpeningHours.order(:id).all
+    @weekdays = ["Sondag","Mandag", "Tirsdag", "Onsdag", "Torsdag", "Fredag", "Lordag"]
```

Comment:
Replace this with time.wday as in http://stackoverflow.com/questions/4844498/rails-3-finding-day-of-week.